<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="InfoBox">
      <property name="title">
       <string>Информация</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="StepLabel">
         <property name="text">
          <string>Шаг : 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ProgressLabel">
         <property name="text">
          <string>Прогресс :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="ProgressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="ControlBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>Панель управления</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="StartBtn">
         <property name="text">
          <string>Пуск</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StopBtn">
         <property name="text">
          <string>Стоп</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StepBtn">
         <property name="text">
          <string>Шаг</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ResetBtn">
         <property name="text">
          <string>Сброс</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NewMazeBtn">
         <property name="text">
          <string>Новый лабиринт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SaveBtn">
         <property name="text">
          <string>Сохранить лабиринт</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
